modules/swift/examples/code_snippets/VectorSearch.swift

```diff
@@ -66,8 +66,9 @@ class VectorSearchSnippets {
         config.numProbes = 8
         config.minTrainingSize = 2500
         config.maxTrainingSize = 5000
-        // end::vs-create-custom-config[]
```

Comment:
The section name, where this tag is used, is [Create a Vector Index](https://preview.docs-test.couchbase.com/cblga/couchbase-lite/current/swift/working-with-vector-search.html#create-a-vector-index), contrary to the tag name. So it somewhat makes sense to have the call for createIndex within it if we go by section name, so I don't mind this change, but this should be reflected for all the other platforms as well for consistency. 

Now, for everything else than iOS - where for Obj-C needs only the tag to be moved, the `createIndex` call doesn't exist at all. So more code changes will be needed. Thus, depending on the urgency, reverting this, Swift only, change might be preferred.